README.md

```diff
@@ -17,7 +17,7 @@ Two main sets of functionalities are provided:
 
 ## `display` command
 
-The CLI command outputs a graphical representation of the graph as a PNG image.
+This CLI command outputs a graphical representation of the whole project knowledge graph, as a PNG image. The graph displayed will be containing also the ontology (if this is loaded via the configuration, as explained later within this document).
```

Comment:
these changes do not correspond to the title. Please either change the title or move changes to another PR.